addons/repair/functions/fnc_getSelectionsToIgnore.sqf

```diff
@@ -63,8 +67,11 @@ private _processedSelections = [];
         continue
     };
 
-    if (_hitpoint select [0,7] isEqualTo "hitera_" || {_hitpoint select [0,8] isEqualTo "hitslat_"} || {_hitpoint select [0,4] isEqualTo "era_"}) then { // skip era/slat
-        TRACE_3("Skipping ERA/Slat HitPoint",_hitpoint,_forEachIndex,_selection);
+    if (_hitpoint select [0,7] isEqualTo "hitera_" || {_hitpoint select [0,4] isEqualTo "era_"}
+    || {_hitpoint select [0,8] isEqualTo "hitslat_"}  || {_hitpoint select [0,5] isEqualTo "slat_"}
+    || {_hitpoint select [0,9] isEqualTo "sideskirt"} || {_hitpoint select [0,6] isEqualTo "armor_"}
+    || {_hitpoint select [0,3] isEqualTo "mud"}) then { // skip era/slat/sideskirt/armor/mudguards
```

Comment:
Hoo boy, I imagine the performance hit is pretty high here ... What about RegEx ?
```suggestion
    if (_hitpoint regexMatch "^(?:(?:(?:hit)?(?:era|slat)|armor)_|sideskirt|mud).*+/o") then { // skip era/slat/sideskirt/armor/mudguards
```

Reply:
4x faster with RegEx (0.0004 ms)
Just to make sure - is this the correct way to check performance?
`["_hitpoint regexMatch '^(?:(?:(?:hit)?(?:era|slat)|armor)_|sideskirt|mud).*+/o'",[{_hitpoint = "hitpoint_test"}],10000] call BIS_fnc_codePerformance;`